src/main/java/frc/robot/commands/GetClosePosition.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.commands;
+import frc.robot.subsystems.GrabberLifter;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class GetClosePosition extends CommandBase {
+    private final GrabberLifter m_GrabberLifter;
+    private final double lowHeight;
+    private final double midHeight;
+    private final double highHeight;
+    private double position;
+
+
+    GetClosePosition(GrabberLifter subsystem) {
+        m_GrabberLifter = subsystem;
+        lowHeight = 200;
+        midHeight = 400;
+        highHeight = 600;
+        position = 0;
+    }
+
+    @Override
+    public void initialize() {
+      position = m_GrabberLifter.getPosOfLift();
+      if(midHeight < position) {
+        if((position-midHeight) < (highHeight-position)) {
+            m_GrabberLifter.lift(-0.3);
+        }
+        else{
+            m_GrabberLifter.lift(0.3);
+        }
+      }
+      else if(lowHeight < position && midHeight > position) {
+        if((position-lowHeight) < (midHeight-position)) {
+            m_GrabberLifter.lift(-0.3);
+        }
+        else{
+            m_GrabberLifter.lift(0.3);
+        }
+      }
+      else if(highHeight < position){
+        m_GrabberLifter.lift(-0.3);
+        }
+  
+  
+  
+    }
+    public void periodic() {
+        position = m_GrabberLifter.getPosOfLift();
```

Comment:
the `execute()` function IS the periodic function in commands. Move everything from inside `periodic()` to `execute()`. Delete the periodic method